modules/nodes-application-secrets-using.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-application-secrets-creating-using-sa_{context}"]
+= Creating and using secrets
+
+As an administrator, you can create a service account token secret. This allows you to distribute a service account token to applications that must authenticate to the API.
+
+.Procedure
+
+. Create a service account in your namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create sa <service_account_name> -n <your_namespace>
+----
+
+. Create a `Secret` object to generate a service account token by running the following command:
++
+[source,yaml]
```

Comment:
AFAIK, `oc apply` command would not be applicable if we are using EOF. Because the file will already be created by using EOF. So `oc apply` in the next step will fail.

Reply:
I will delete the EOF and add the next step as `oc apply`

Reply:
Ah sorry, I meant to remove the EOF element from my example. Here is the correct example:

```
. Save the following YAML example to a file named `service-account-token-secret.yaml`. The example includes a `Secret` object configuration that you can use to generate a service account token:
+
[source,yaml]
----
apiVersion: v1
kind: Secret
metadata:
  name: <secret_name> <1>
  annotations:
    kubernetes.io/service-account.name: "sa-name" <2>
type: kubernetes.io/service-account-token <3>
----
<1> Specifies a secret name.
<2> Specifies an existing service account name. If you are creating both the `ServiceAccount` and the `Secret` objects, create the `ServiceAccount` object first.
<3> Specifies a service account token secret.

. Generate the service account token by applying the file:
+
[source,terminal]
----
$ oc apply -f service-account-token-secret.yaml
----
```

Reply:
Addressed